html-block-and-inline/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline </title>
+  </head>
+  <body>
+    <div id="Snuggie">
+      <h1><b>Blue Americana Snuggie&reg; Blanket</b></h1>
```

Comment:
you don't need to use the `b` tag within an `h1` or `h2` tag since they have bold styling by default.